docs/agent-reference/how-to-run-agent.md

```diff
@@ -8,12 +8,24 @@ AducIotAgent '<IoT device connection string>' [options...]
 
 ### Additional Arguments
 
-#### --EnableIotHubTracing
+#### --version
+
+With this option enabled, the reference agent will output the version.
+
+#### --enable-iothub-tracing
```

Comment:
super nit: is there a way to make it clear that these descriptions are associated with the text above? 

Maybe instead of making them h3s we include them as  a code block inline e.g.

--enable-iothub-tracing tells the reference agent to output verbose tracing from the Azure IoT C SDK. This option is useful for troubleshooting connection issues.